src/test/java/com/checkout/accounts/AccountsClientImplTest.java

```diff
@@ -181,6 +181,21 @@ void shouldCreatePaymentInstrument() throws ExecutionException, InterruptedExcep
         assertEquals(response, future.get());
     }
 
+    @Test
+    void shouldUpdatePaymentInstrument() throws ExecutionException, InterruptedException {
+
+        final IdResponse response = mock(IdResponse.class);
+        final UpdatePaymentInstrumentRequest request = mock(UpdatePaymentInstrumentRequest.class);
+
+        when(apiClient.patchAsync(eq("accounts/entities/entity_id/payment-instruments/instrument_id"), eq(authorization), eq(IdResponse.class), any(UpdatePaymentInstrumentRequest.class), isNull()))
+                .thenReturn(CompletableFuture.completedFuture(response));
+
+        final CompletableFuture<IdResponse> future = accountsClient.updatePaymentInstrumentDetails("entity_id", "instrument_id", request);
+
+        assertNotNull(future.get());
+        assertEquals(response, future.get());
+    }
+
```

Comment:
Can we test 422 and 401 response too?

Reply:
Usually we follow that pattern on IT on this SDK. for mocking purposes, not sure if make any sense.

Reply:
👍